studio/app/optinist/wrappers/caiman/cnmf.py

```diff
@@ -116,9 +118,12 @@ def caiman_cnmf(
         backend="local", n_processes=None, single_thread=True
     )
 
-    cnm = cnmf.CNMF(n_processes=n_processes, dview=dview, Ain=None, params=ops)
+    cnm = cnmf.CNMF(n_processes=n_processes, dview=dview, Ain=Ain, params=ops)
```

Comment:
@quanpython 
The following warning occurs when the process is executed.

> WARNING:root:No parameter do_refit found!
> WARNING:root:No parameter thr found!

If it is possible to resolve the warning, please do so.

Reply:
Fix confirmed.